compute/cker/src/DepthwiseConv.test.cc

```diff
@@ -0,0 +1,320 @@
+  }
+}
+
+TEST(CKer_Operation, neg_DepthwiseConv)
+{
+  // Not supported Dilation
+  {
+    nnfw::cker::DepthwiseConvParams params{};
+    params.padding_type = nnfw::cker::PaddingType::kSame;
+    params.padding_values.width = 0;
+    params.padding_values.height = 1;
+    params.stride_width = 1;
+    params.stride_height = 1;
+    params.dilation_width_factor = 2;
+    params.dilation_height_factor = 2;
+    params.depth_multiplier = 1;
+    params.float_activation_min = std::numeric_limits<float>::lowest();
+    params.float_activation_max = std::numeric_limits<float>::max();
+
+    nnfw::cker::Shape input_shape{1, 6, 6, 1}; // n, h, w, c
+    std::vector<float> input = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                                0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0,
+                                0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
+    nnfw::cker::Shape filter_shape{1, 2, 2, 1}; // 1, h, w, c
+    std::vector<float> filter = {1.0, 2.0, 3.0, 4.0};
+    nnfw::cker::Shape bias_shape{1};
+    std::vector<float> bias = {0.0};
+    nnfw::cker::Shape output_shape{1, 3, 3, 1}; // n, h, w, c
+    std::vector<float> expected = {4.0, 0.0, 3.0, 0.0, 0.0, 0.0, 2.0, 0.0, 1.0};
+
+    DepthwiseConvVerifier<float> verifier;
+    verifier.prepare(output_shape, filter_shape);
+    verifier.checkException(params, input_shape, input.data(), filter_shape, filter.data(),
+                            bias_shape, bias.data(), output_shape, expected.data());
+  }
+
+  // No bias data
+  {
+    nnfw::cker::DepthwiseConvParams params{};
+    params.padding_type = nnfw::cker::PaddingType::kSame;
+    params.padding_values.width = 0;
+    params.padding_values.height = 1;
+    params.stride_width = 1;
+    params.stride_height = 1;
+    params.dilation_width_factor = 1;
+    params.dilation_height_factor = 1;
+    params.depth_multiplier = 1;
+    params.float_activation_min = std::numeric_limits<float>::lowest();
+    params.float_activation_max = std::numeric_limits<float>::max();
+
+    nnfw::cker::Shape input_shape{1, 6, 6, 1}; // n, h, w, c
+    std::vector<float> input = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                                0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0,
+                                0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
+    nnfw::cker::Shape filter_shape{1, 2, 2, 1}; // 1, h, w, c
+    std::vector<float> filter = {1.0, 2.0, 3.0, 4.0};
+    nnfw::cker::Shape bias_shape{1};
+    nnfw::cker::Shape output_shape{1, 3, 3, 1}; // n, h, w, c
+    std::vector<float> expected = {4.0, 0.0, 3.0, 0.0, 0.0, 0.0, 2.0, 0.0, 1.0};
+
+    DepthwiseConvVerifier<float> verifier;
+    verifier.prepare(output_shape, filter_shape);
+    verifier.checkDeath(params, input_shape, input.data(), filter_shape, filter.data(), bias_shape,
+                        nullptr, output_shape, expected.data());
```

Comment:
The DepthwiseConv2D operation always has a `bias_data`, so I removed the `if(bias_data != nullptr)` check case. Instead, I add a new test case to check if app makes an assertion if the bias_data is null.

Reply:
What I meant was that it would be better to add a test to check if optional bias is supported. But I think it's not bad to add this test instead of my intention.

Reply:
there is no code that access bias_data, and it is passed as argument , there is no need to check for nullptr. 
of course, even when there is no explicit nullptr input, sometimes the static analysis tool judges it to be an error when accessing the variable without checking for nullptr. of course, this does not apply in this case.
so, i think it is a good choice.

Reply:
I'm sorry but I couldn't catch your intention. :(

> add a test to check if optional bias is supported

Could you explain more about this test case?

Reply:
> Could you explain more about this test case?

`biasHelper` abort if `bias_data` is nullptr. It means that optional bias(`nullptr`) is not allowed for DepthwiseConv2D.
But what I meant was that optional bias is allowed and the test verifies whether the DepthwiseConv2D kernel works well with optional bias(`nullptr`).